### ISSUE TITLE
Feature request - export layers with settings

### ISSUE THREAD

**Issue by arxeiss** [photopea/photopea#1685]
It is great I can **Export layers**, it is amazing. But I cannot set anything there.

For example when exporting PNG, I want to not use palette. And I need to export each layer in single because I cannot set this in Export layers. Would it be possible to set those additional export settings in this feature too?

Thank you

**Comment by photopea** [comment]
Hi, Export layers is a very simple feature right now.

Why don't you want to use palettes? The color information is still the same (no quality loss), but PNG files get smaller.

**Comment by arxeiss** [comment]
The feature request was more general, like also set quality, also with JPG etc.

But I need to export files without palette because I was creating icons into Sublime Text Plugin and it doesn't work with palette.

**Comment by photopea** [comment]
Ok, I can add such format settings to Export Layers feature.

But is it ok if I allow just one format for export? Now, each layer can be exported in 3 formats (PNG, JPG, SVG) with one click. Adding the options for all three formats next to each other would make the UI quite complex.

**Comment by arxeiss** [comment]
I think it is up to you. It would be nice to have those options there. But maybe others prefer to export into multiple formats rather than have those options.

**Comment by photopea** [comment]
Hi, I added export options to a Export Layers window. They are the same as in Save For Web.

However, I don't know if they are useful, because you can see the quality and the file size only after exporting.
